### PR TITLE
change code standard to psr-2

### PR DESCRIPTION
**Pull request by stojg**

### CODE REVIEW

File: .php_cs.cache [hunk 1]
```diff
@@ -0,0 +1 @@
...rue,"array_syntax":{"syntax":"short"},"heredoc_to_nowdoc":true,"no_short_echo_tag":true,"no_useless_else":true,"no_useless_return":true,"ordered_class_elements":true,"ordered_imports":true,"phpdoc_add_missing_param_annotation":true,"phpdoc_order":true,"semicolon_after_instruction":true},"hashes":{"src\/\/SilverStripe\/BlowGun\/Command\/BaseCommand.php":1942505830,"src\/\/SilverStripe\/BlowGun\/Command\/ListenCommand.php":1331485007,"src\/\/SilverStripe\/BlowGun\/Credentials\/BlowGunCredentials.php":3684517042,"src\/\/SilverStripe\/BlowGun\/Exceptions\/MessageLoadingException.php":3649150853,"src\/\/SilverStripe\/BlowGun\/Model\/Command.php":1926045844,"src\/\/SilverStripe\/BlowGun\/Model\/Message.php":3068534860,"src\/\/SilverStripe\/BlowGun\/Model\/Status.php":788797574,"src\/\/SilverStripe\/BlowGun\/Service\/SQSHandler.php":4060045931,"src\/\/SilverStripe\/BlowGun\/Tests\/CommandTest.php":3784959247,"src\/\/SilverStripe\/BlowGun\/Tests\/Helpers\/MockClient.php":3520127544,"src\/\/SilverStripe\/BlowGun\/Tests\/Helpers\/MockHandler.php":3296674075,"src\/\/SilverStripe\/BlowGun\/Tests\/MessageTest.php":3100089502,"src\/\/SilverStripe\/BlowGun\/Tests\/SQSHandlerTest.php":1505331025}}
```
Comment (halkyon) [inline .php_cs.cache]: chuck this in `.gitignore`

Comment (stojg) [reply .php_cs.cache]: ignored

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[{*.js,*.jsx}]
```
Comment (halkyon) [inline .editorconfig]: irrelevant, remove

File: .editorconfig [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+indent_size = 4
+indent_style = tab
+
+[*.md]
+trim_trailing_whitespace = false
+
+[*.ss]
```
Comment (halkyon) [inline .editorconfig]: irrelevant, remove

File: .editorconfig [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+indent_style = tab
+indent_size = 4
+
+[*.yml]
+indent_size = 2
+indent_style = space
+
+[{.travis.yml,package.json}]
```
Comment (halkyon) [inline .editorconfig]: irrelevant, remove

File: .editorconfig [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+# The indent size used in the `package.json` file cannot be changed
+# https://github.com/npm/npm/pull/3180#issuecomment-16336516
+indent_size = 2
+indent_style = space
+
+[Makefile]
```
Comment (halkyon) [inline .editorconfig]: irrelevant, remove

**Comment by halkyon** [comment]
can you reformat `composer.json` as well to be consistent with the `.editorconfig`?

**Comment by stojg** [comment]
sure

**Comment by halkyon** [comment]
you missed `config.php` in the root :D

**Comment by stojg** [comment]
Finally!

**Pull request merged**